draft-ietf-teep-protocol.md

```diff
@@ -597,6 +597,200 @@ done by the TEE independent of whether TEEP is used or some other update
 mechanism.
 See section 5 of {{I-D.ietf-teep-architecture}} for further discussion.
 
+
+
+The Update Message has a SUIT_Envelope containing SUIT manifests. Following are some SUIT manifest examples in the messages.
+
+### Example 1: Having one SUIT Manifest pointing to a URI of a Trusted Component Binary
+
+This subsection shows a SUIT Manifest example that has a URI pointing to a Trusted Component Binary.
+
+A Trusted Component Developer creates a new Trusted Component Binary and generates an associated SUIT manifest with the filename "tc-uuid.suit". The filename "tc-uuid.suit" is used in Example 3 later. Then the Trusted Component Developer hosts the Trusted Component Binary at a Trusted Component Developers' URI. The TAM always receives the latest SUIT manifest from the Trusted Component Developer.
+
+Trusted Component Developers provide the delivery point of the Trusted Component Binary associated with the URI in the SUIT manifest and the URI will not be changeable by the TAM since the SUIT manifest is signed by Trusted Component Developers.
+
+
+Pros:
+
+ - The Trusted Component Developer can ensure that the intact Trusted Component Binary is downloaded by TEEP Devices
+ - The TAM does not have to deliver Update message containing Trusted Component Binary which may have a large size
+
+Cons:
+
+ - The Trusted Component Developer must host the Trusted Component Binary server
+ - The TEEP Device must fetch the Trusted Component Binary in another connection after receiving an Update message
+
+~~~~
+    +------------+           +-------------+
+    | TAM        |           | TEEP Agent  |
+    +------------+           +-------------+
+
+             Update  ---->
+
+    +=================== teep-protocol(TAM) ==================+
+    | TEEP_Message([                                          |
+    |   TEEP-TYPE-update,                                     |
+    |   options: {                                            |
+    |     manifest-list: [                                    |
+    |       += suit-manifest "tc-uuid.suit" (TC Developer) =+ |
+    |       | SUIT_Envelope({                               | |
+    |       |   manifest: {                                 | |
+    |       |     install: {                                | |
+    |       |       set-parameter: {                        | |
+    |       |         uri: "https://tc.org/tc-uuid.ta"      | |
```

Comment:
tc.org is not a reserved domain name.  These should all be replaced with example.org per RFC 6761